.github/workflows/build.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build backend
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - 'v*'
+    paths:
+      - 'expense-backend/**'
```

Comment:
With only one CI file, that is building the backend and the frontend, do we need these?